README.md

```diff
@@ -1,12 +1,11 @@
-packager
-========
+# GOV.UK packaging
 
-This repository contains some scripts to aid the building of debian source
-packages (.dsc) files for submission to Launchpad or another build server for
-building into binary debian packages (.deb)
+This repo creates packages for GOV.UK.
 
-Usage
------
+## fpm
 
-See the ["Getting Started" documentation at
-`doc/getting_started.md`](doc/getting_started.md).
+This is the way we create packages now. It is easier than using debuild.
```

Comment:
I'm not sure what I'd say for that except that the `debuild` stuff has 4 Markdown files worth of documentation (which I think it needs) but fpm doesn't require that much explaining because it's so much more straightforward.